packages/plugins/plugin-table/src/model/table-model.test.ts

```diff
@@ -130,7 +130,8 @@ describe('TableModel', () => {
           ],
         }));
 
-        model = new TableModel({ table: table as any, data });
+        model = new TableModel({ table: table as any });
```

Comment:
pls. try to remove otherwise it invalidates the test.

Reply:
Elaborate?